### PR TITLE
Complete / enrich readme files for Spring Cloud integration with Service Bus

### PR DESCRIPTION
**Pull request by saragluna**
Some of the README files were not complete when migrated to this repo. This PR is to add content to them. Projects including:

- spring-integration-service-bus
- spring-cloud-starter-service-bus
- spring-cloud-stream-binder-service-bus-core
- spring-cloud-stream-binder-service-bus-queue
- spring-cloud-stream-binder-service-bus-topic

### CODE REVIEW

File: sdk/spring/azure-spring-cloud-servicebus-stream-binder-core/README.md [hunk 1]
```diff
@@ -1,93 +1,81 @@
...
 
-It supports the following configurations with the format of `spring.cloud.stream.servicebus.queue.bindings.<channelName>.producer`.
+This implementation uses [Spring Integration][spring_integration] Service Bus Channel Adapters at its foundation. 
 
-**_sync_**
+Please refer to [Service Bus Queue Binder][service_bus_queue_binder] and [Service Bus Topic Binder][service_bus_topic_binder] for more details.
 
-Whether the producer should act in a synchronous manner with respect to writing records into a stream. If true, the 
-producer will wait for a response after a send operation.
+## Examples
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-cloud-servicebus-stream-binder-core/README.md:34]: we can add links to associated samples

File: sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md [hunk 2]
```diff
@@ -1,63 +1,52 @@
...
 
-## Key concepts
+[Package (Maven)][package] | [API reference documentation][refdocs] | [Product documentation][docs] | [Samples][sample]
+
+## Getting started
+
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven](http://maven.apache.org/) 3.0 and above
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md:13]: same here

File: sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md [hunk 3]
```diff
@@ -1,63 +1,52 @@
-# Spring Cloud Azure Service Bus Topic Stream Binder client library for Java
+# Azure Service Bus Topic Spring Cloud Stream Binder client library for Java
 
 The project provides **Spring Cloud Stream Binder for Azure Service Bus Topic** which allows you to build message-driven 
-microservice using **Spring Cloud Stream** based on [Azure Service Bus Topic](https://azure.microsoft.com/en-us/services/service-bus/) service.
+microservice using **Spring Cloud Stream** based on [Azure Service Bus Topic][azure_service_bus].
 
-## Key concepts
+[Package (Maven)][package] | [API reference documentation][refdocs] | [Product documentation][docs] | [Samples][sample]
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md:6]: same here

File: sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md [hunk 4]
```diff
@@ -115,6 +104,53 @@ Controls if is a session aware consumer. Set it to `true` if is a topic with ses
 
 Default: `false`
 
+## Examples 
+
+Please use this `sample` as a reference
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md:109]: same here, add a link

File: sdk/spring/azure-spring-cloud-starter-servicebus/README.md [hunk 5]
```diff
@@ -1,8 +1,70 @@
...
+
+[Package (Maven)][package] | [API reference documentation][refdocs] | [Samples][sample]
 
-## Key concepts
 ## Getting started
+
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven](http://maven.apache.org/) 3.0 and above
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-cloud-starter-servicebus/README.md:12]: same here

File: sdk/spring/azure-spring-cloud-starter-servicebus/README.md [hunk 6]
```diff
@@ -1,8 +1,70 @@
...
+```
+[//]: # ({x-version-update-end})
+
+
+## Key concepts
+[Spring Integration][spring_integration] enables lightweight messaging within Spring-based applications and supports integration with external systems via declarative adapters.
+
+This project provides Spring Integration adaption with Azure Service Bus and the ability to auto-configure connection to Azure Service Bus.
+
 ## Examples
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-cloud-starter-servicebus/README.md:31]: same here, we can add a link

File: sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md [hunk 7]
```diff
@@ -115,6 +104,53 @@ Controls if is a session aware consumer. Set it to `true` if is a topic with ses
...
+Please follow [instructions here][contributing_md] to build from source or contribute.
+
+<!-- Links -->
+[azure_service_bus]: https://azure.microsoft.com/services/service-bus/
+[azure_subscription]: https://azure.microsoft.com/free
+[contributing_md]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/CONTRIBUTING.md
+[docs]: https://docs.microsoft.com/azure/developer/java/spring-framework/configure-spring-cloud-stream-binder-java-app-with-service-bus
+[package]: https://mvnrepository.com/artifact/com.microsoft.azure/spring-cloud-azure-servicebus-topic-stream-binder
+[refdocs]: https://azure.github.io/azure-sdk-for-java/spring.html#https://azure.github.io/azure-sdk-for-java/spring.html#spring-cloud-azure-servicebus-topic-stream-binder
+[sample]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md:149]: seems we don't need it any more, we can remove it.

File: sdk/spring/azure-spring-cloud-starter-servicebus/README.md [hunk 8]
```diff
@@ -1,8 +1,70 @@
...
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+Please follow [instructions here][contributing_md] to build from source or contribute.
+
+<!-- Links -->
+[azure_subscription]: https://azure.microsoft.com/free
+[contributing_md]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/CONTRIBUTING.md
+[package]: https://mvnrepository.com/artifact/com.microsoft.azure/spring-cloud-starter-azure-servicebus
+[refdocs]: https://azure.github.io/azure-sdk-for-java/spring.html#spring-cloud-starter-azure-servicebus
+[sample]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-cloud-starter-servicebus/README.md:66]: same here

File: sdk/spring/azure-spring-integration-servicebus/README.md [hunk 9]
```diff
@@ -1,9 +1,72 @@
-# Azure Spring Integration service bus client library for Java
+# Azure Service Bus Spring Integration client library for Java
+
+The Spring Integration for Azure Service Bus extension project provides inbound and outbound channel adapters for Azure Service Bus. 
+Microsoft Azure Service Bus is a fully managed enterprise integration message broker. Service Bus can decouple applications and services. 
+Service Bus offers a reliable and secure platform for asynchronous transfer of data and state. 
+
+[Package (Maven)][package] | [API reference documentation][refdocs] | [Samples][sample]
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-integration-servicebus/README.md:7]: same here, add the source code tag.

File: sdk/spring/azure-spring-integration-servicebus/README.md [hunk 10]
```diff
@@ -1,9 +1,72 @@
...
 
-## Key concepts
 ## Getting started
+
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven](http://maven.apache.org/) 3.0 and above
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-integration-servicebus/README.md:14]: same here

File: sdk/spring/azure-spring-integration-servicebus/README.md [hunk 11]
```diff
@@ -1,9 +1,72 @@
...
+[//]: # ({x-version-update-end})
+
+
 ## Key concepts
+[Spring Integration][spring_integration] enables lightweight messaging within Spring-based applications and supports integration with external systems via declarative adapters.
+
+This project provides inbound and outbound channel adapters for Azure Service Bus.
+
 ## Examples
+
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-integration-servicebus/README.md:34]: and here

File: sdk/spring/azure-spring-integration-servicebus/README.md [hunk 12]
```diff
@@ -1,9 +1,72 @@
...
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+Please follow [instructions here][contributing_md] to build from source or contribute.
+
+<!-- Links -->
+[azure_subscription]: https://azure.microsoft.com/free
+[contributing_md]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/CONTRIBUTING.md
+[package]: https://mvnrepository.com/artifact/com.microsoft.azure/spring-integration-servicebus
+[refdocs]: https://azure.github.io/azure-sdk-for-java/spring.html#spring-integration-servicebus
+[sample]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-integration-servicebus/README.md:68]: and here

File: sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md [hunk 13]
```diff
@@ -1,60 +1,49 @@
...
-## Key concepts
+[Package (Maven)][package] | [API reference documentation][refdocs] | [Product documentation][docs] | [Samples][sample]
+
+## Getting started
 
-### Service Bus Queue Binder Overview
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven](http://maven.apache.org/) 3.0 and above
```
[the remark this answers is not in the document]
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md:13]: Good point.

File: sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md [hunk 14]
```diff
@@ -1,60 +1,49 @@
-# Spring Cloud Azure Service Bus Queue Stream Binder client library for Java
+# Azure Service Bus Queue Spring Cloud Stream Binder client library for Java
 
 The project provides **Spring Cloud Stream Binder for Azure Service Bus Queue** which allows you to build message-driven 
-microservice using **Spring Cloud Stream** based on [Azure Service Bus Queue](https://azure.microsoft.com/en-us/services/service-bus/) service.
+microservice using **Spring Cloud Stream** based on [Azure Service Bus Queue][azure_service_bus].
 
-## Key concepts
+[Package (Maven)][package] | [API reference documentation][refdocs] | [Product documentation][docs] | [Samples][sample]
```
[the remark this answers is not in the document]
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md:6]: Added.

File: sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md [hunk 15]
```diff
@@ -112,6 +101,52 @@ Controls if is a session aware consumer. Set it to `true` if is a queue with ses
 
 Default: `false`
 
+## Examples
+
+Please use this `sample` as a reference for how to use this binder in your projects.
```
[the remark this answers is not in the document]
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md:106]: Added.

File: sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md [hunk 16]
```diff
@@ -112,6 +101,52 @@ Controls if is a session aware consumer. Set it to `true` if is a queue with ses
...
+
+Please follow [instructions here][contributing_md] to build from source or contribute.
+
+<!-- Links -->
+[azure_service_bus]: https://azure.microsoft.com/services/service-bus/
+[azure_subscription]: https://azure.microsoft.com/free
+[contributing_md]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/CONTRIBUTING.md
+[docs]: https://docs.microsoft.com/azure/developer/java/spring-framework/configure-spring-cloud-stream-binder-java-app-with-service-bus
+[package]: https://mvnrepository.com/artifact/com.microsoft.azure/spring-cloud-azure-servicebus-queue-stream-binder
+[refdocs]: https://azure.github.io/azure-sdk-for-java/spring.html#https://azure.github.io/azure-sdk-for-java/spring.html#spring-cloud-azure-servicebus-queue-stream-binder
```
[the remark this answers is not in the document]
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md:144]: fixed

File: sdk/spring/azure-spring-cloud-servicebus-stream-binder-core/README.md [hunk 17]
```diff
@@ -1,93 +1,81 @@
...
-Please use this `sample` as a reference for how to use this binder in your projects. 
-
-### Feature List 
-
-- [Dependency Management](#dependency-management)
-- [Configuration Options](#configuration-options)
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven](http://maven.apache.org/) 3.0 and above
```
[the remark this answers is not in the document]
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-servicebus-stream-binder-core/README.md:13]: fixed

File: sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md (hunk 2 above)
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md:13]: fixed

File: sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md (hunk 3 above)
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md:6]: fixed

File: sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md (hunk 4 above)
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md:109]: fixed

File: sdk/spring/azure-spring-cloud-starter-servicebus/README.md (hunk 5 above)
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-starter-servicebus/README.md:12]: fixed

File: sdk/spring/azure-spring-cloud-starter-servicebus/README.md (hunk 8 above)
Comment (saragluna) [reply sdk/spring/azure-spring-cloud-starter-servicebus/README.md:66]: Why do we remove these sample links?

File: sdk/spring/azure-spring-integration-servicebus/README.md (hunk 9 above)
Comment (saragluna) [reply sdk/spring/azure-spring-integration-servicebus/README.md:7]: fixed

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**